msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -35,19 +36,19 @@ Visit the [Teams store](https://go.microsoft.com/fwlink/p/?LinkID=2183121) and e
 >
 > * Currently when a third person is added to a one-on-one call, the call is elevated to a group call that means a new session starts. Apps added to the one-on-one call are not available in the group call. However, they can be added again.
 >
-> * Currently app experiences are not supported in Teams channel meetings (both scheduled meetings and instant meetings).
+> * Currently app experiences are supported in Teams public scheduled channel meetings only. Adding an app to a channel or team does not add app to the channel meeting and vice versa.
```

Comment:
> Currently app experiences are supported in Teams public scheduled channel meetings only. 

This implies that there are scenarios that don't support apps, but we don't say what they are. Could we add something like "Apps are not yet available in X, Y, and Z". Let's not leave users to guess.

Reply:
@v-ksar please change the last bullet point to say " Currently app experiences are not supported in Teams instant channel meetings."

Reply:
Updated as mentioned.